swagger/v5/swagger.yaml

```diff
@@ -837,6 +837,409 @@ paths:
                               description: Amount of payment made
                               example: 101.01
         required: true
+  "/assessments/{assessment_id}/partner_financials":
+    post:
+      summary: 'create '
+      tags:
+      - Assessment components
+      description: Adds details of an applicant's partner.
```

Comment:
Bit confused here - isn't this file generated? Should it really be checked in to source control?

Reply:
@starswan it is generated by a rake task that doesn't happen as part of the deploy pipeline. In fact Circle has a job called check_swagger that fails out if this hasn't been generated when the specs change.

Reply:
fair enough @patrick-laa